external/model-preparation-algorithm/mpa_tasks/apis/classification/task.py

```diff
@@ -280,11 +280,17 @@ def _add_predictions_to_dataset(self, prediction_results, dataset, update_progre
 
     def _init_recipe_hparam(self) -> dict:
         warmup_iters = int(self._hyperparams.learning_parameters.learning_rate_warmup_iters)
-        lr_config = (
-            ConfigDict(warmup_iters=warmup_iters)
-            if warmup_iters > 0
-            else ConfigDict(warmup_iters=warmup_iters, warmup=None)
-        )
+        if self._multilabel:
+            # hack to use 1cycle policy
+            lr_config = (
+                ConfigDict(max_lr=self._hyperparams.learning_parameters.learning_rate, warmup=None)
```

Comment:
I can't find update of learning rate for multi-label. Is it necessary now? Are you going to update soon?

Reply:
This is parameter `max_lr` needed for 1cycle policy. It set to current learning rate. Unfortunately, there is no other way to receive information about learning rate, but we need it to scheduler to work.
For now, there is no learning rate update. In the nearest future we will be dividing templates, but now we have unite learning rate parameter for both tasks.

Reply:
It worth's noticing that now we can't change the scheduler for multilabel classification without changing this line because the other schedulers don't expect `max_lr` parameter. We can workaround this by letting each scheduler ignore irrelevant arguments, otherwise we have to change the overall design of the recipies initialization step to fix the mentioned problem.

Reply:
Thank you for letting me know. I expect to resolve this after separating the templates.